frontend/src/pages/restartPage/moveAwayFromApDialog/MoveAwayFromApDialogContainer.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import MoveAwayFromApDialog from "./MoveAwayFromApDialog";
+
+export type Props = {
+  active: boolean;
+  piTopIpAddress: string;
+  onSkip: () => void;
+};
+
+export default ( props: Props) => {
+  return (
+    <MoveAwayFromApDialog
+      {...props}
```

Comment:
the container component can be skipped if is is pure ui component with no logic, seems to be in this case.

in fact it sort of breaks the 'reusable ui component' pattern if it renders a (presumably non-reusable) container (as RestartPage does in this case). this isn't super important, something we're really only recently refined in Further, but a better pattern if a container is required as a child of a ui component is to pass it in as a prop (known as a [slot](https://github.com/pi-top/further-frontend/blob/15a061628295d88d579f5765aedc2cecf6549777/dev-notes/component-structure.md))